scalameta/trees/shared/src/main/scala/scala/meta/internal/trees/ParentChecks.scala

```diff
@@ -107,4 +107,13 @@ object ParentChecks {
   def TypeMethod(tree: Type.Method, parent: Tree, destination: String): Boolean = {
     parent.is[Type] || parent.is[Defn.Type]
   }
+
+  def TermBlock(tree: Term.Block, parent: Tree, dest: String): Boolean = {
+    val isExportOK = parent.is[Defn.ExtensionGroup]
+    tree.stats.forall {
+      case x if x.isBlockStat => x.isBlockStat
```

Comment:
it's always true on the right, no?

also, forall will now throw a match exception instead of returning false.